src/util/string.cpp

```diff
@@ -1072,16 +1072,17 @@ std::optional<v3f> str_to_v3f(std::string_view str)
 {
     str = trim(str);
 
-    if (str.empty()) {
+    if (str.empty())
         return std::nullopt;
-    }
 
     // Strip parentheses if they exist
     if (str.front() == '(' && str.back() == ')') {
         str.remove_prefix(1);
         str.remove_suffix(1);
     }
 
+    str = trim(str);
```

Comment:
Nitpick: I'd move this inside the "strip parenthesis" if body, makes the logic more obvious IMO.